src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Call.cs

```diff
@@ -939,6 +940,11 @@ private static ImmutableArray<RefKind> GetEffectiveArgumentRefKinds(ImmutableArr
                     fillRefKindsBuilder(argumentRefKindsOpt, parameters, ref refKindsBuilder);
                     refKindsBuilder[i] = argRefKind == RefKind.None ? paramRefKind : RefKindExtensions.StrictIn;
                 }
+                else if (paramRefKind == RefKind.RefReadOnlyParameter)
+                {
+                    fillRefKindsBuilder(argumentRefKindsOpt, parameters, ref refKindsBuilder);
+                    refKindsBuilder[i] = RefKind.RefReadOnlyParameter;
```

Comment:
>RefKind.RefReadOnlyParameter [](http://example.com/codeflow?start=41&length=28)

Should we have a "strict" version similar to `in`?

Reply:
I'm not sure, will investigate, perhaps add it in a follow up PR. Adding a prototype comment for now.

Reply:
From my investigation, "strict `in`" is only used to pass strict readonly address kind here:

https://github.com/dotnet/roslyn/blob/d0f4099ffa7f8ff31fd397a4a96f404e1cd7dfc7/src/Compilers/CSharp/Portable/CodeGen/EmitExpression.cs#L717

which only makes a difference for field referenes in a PEVerify compat mode:

https://github.com/dotnet/roslyn/blob/d0f4099ffa7f8ff31fd397a4a96f404e1cd7dfc7/src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs#L4773-L4783

Not sure if we need to support that for `ref readonly` parameters.

I've only looked through references to `RefKindExtensions.StrictIn`, not sure if I should look elsewhere.

Reply:
We are reusing ```RefKind.In``` and ```RefKindExtensions.StrictIn``` now.